kairos-cli/src/crypto/signer.rs

```diff
@@ -0,0 +1,38 @@
+use std::path::Path;
+
+use super::private_key::CasperPrivateKey;
+use super::public_key::CasperPublicKey;
+use crate::crypto::error::CryptoError;
+use casper_types::bytesrepr::ToBytes;
+use casper_types::{crypto, PublicKey};
+
+pub struct CasperSigner {
+    secret_key: CasperPrivateKey,
+    pub public_key: CasperPublicKey,
+}
+
+#[allow(unused)]
+impl CasperSigner {
+    pub fn from_file<P: AsRef<Path>>(file_path: P) -> Result<Self, CryptoError> {
```

Comment:
Same as above

Reply:
Fixed in https://github.com/cspr-rad/kairos/commit/55560461a0781fef8734bdd43b0118fb740ea9ec.